src/electionguard/schema.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Tuple
+from os.path import join, dirname, realpath
+from json import load
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+
+__all__ = ["election_description_schema", "validate_json_schema"]
+
+
+def _load_schema(json_schema_file_name: str) -> Any:
+    """Loads the given schema"""
+    with open(join(dirname(realpath(__file__)), json_schema_file_name), "r") as file:
+        schema = load(file)
+    return schema
+
+
+election_description_schema = _load_schema("election_description_schema.json")
```

Comment:
I'm not 100% confident of this, but I feel like this shouldn't be run automatically on import like this, i.e. any code that even imports this module will wait for the schema to load during import.

Would putting it behind a function be better?

Reply:
I originally had something like this. I thought it made it more convenient to have it like this. I originally had a get_election_description_schema. I can reimplement quickly.